src/client/index.tsx

```diff
@@ -10,7 +10,8 @@ import { Provider } from "react-redux";
 import { BrowserRouter } from "react-router-dom";
 import { Action, createStore } from "redux";
 import App from "../shared/App";
-import "./assets/App.scss";
+
+import "../shared/components/styles/core/index.scss";
```

Comment:
I know we use the index.* for registering modules but in this case this files does more. Should we name it something more descriptive?

Reply:
We could name it `core.scss`, but is that duplicative since that's the dir name too? I guess that's probably the right way to go since it's how were handling individual components.

Reply:
Yeah not a big deal since we can always change it later. I'm thinking entry points should all folow the same pattern eventually.